src/init.js

```diff
@@ -0,0 +1,75 @@
+/*------------------------------------------------------------------------------------------------\
+  Initialize adverse event explorer.
+\------------------------------------------------------------------------------------------------*/
+
+export function init(data) {
+    var element = d3.select(this.element);
```

Comment:
or perhaps we should just follow the webcharts paradigm and go with `wrap` - I also want to bind the raw data to the root object. Going to cut this code review short and make some commits.